examples/extraction.py

```diff
@@ -1,6 +1,6 @@
 from mwapi import Session
 from revscoring.extractors import api
-from revscoring.features import temporal, wikitext
+from revscoring.features import wikitext, temporal
```

Comment:
Imports should be in sort order.  See https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement for a discussion.